src/components/ResourceBrowser/ResourceList/ResourceList.tsx

```diff
@@ -254,6 +254,23 @@ export default function ResourceList() {
         }
     }
 
+    const secondsParser = (duration: any): number => {
+        //Parses time(format:- ex. 4h20m) in second
+        let totalTimeInSec: number = 0
```

Comment:
Bug :
what if value if 2d3h5m3s (you missed day).

Can we use a proper format here instead of writing our own logic?

Reply:
Value can come in 2d3h4m5s (and day is optional and so on...)

Reply:
bug : secondsParser("4h3m5s") -> your code is returning 14408. it should be 14585

Reply:
We should move this block to somewhere common place so that we can reuse the function